EIPS/eip-5539.md

```diff
@@ -0,0 +1,232 @@
+---
+eip: 5539
+title: Revocation List Registry
+description: Registry of revocation lists for revoking arbitrary data.
+author: Philipp Bolte (@strumswell), Lauritz Leifermann (@lleifermann), Dennis von der Bey (@DennisVonDerBey)
+discussions-to: https://ethereum-magicians.org/t/eip-5539-revocation-list-registry/10573
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-26
+requires: 712
+---
+
+## Abstract
+This EIP proposes a set of methods and standards for an RBAC-enabled registry of indicators aimed for usage in revocations.
```

Comment:
[Editorial] consider spell out `RBAC` at its first occurance

Reply:
I strongly agree here. Acronyms and jargon should be defined unless well-known.

Reply:
I simplified it.